mozmeao/trafficcop
# Import CookieHelper library from NPM (Fixes #7)

- Adds a webpack config for the demo pages to load `@mozmeao/cookie-helper` from NPM.
- Updates documentation to include info about the cookie helper dependency.
- Removes duplicated cookie helper lib source code from the repo.

To test:

1. `cd demo`
2. `npm install`
3. `npm start`
4. Open http://localhost:3030/
5. Ensure the traffic cop example still work correctly.
